### ISSUE TITLE
cluster/kube-up.sh failed

### ISSUE THREAD

**Issue by ivan-exness** [kubernetes/kubernetes#22700]
I have import vmdk successfully
start ssh-agent
```
 eval "$(ssh-agent)"
```

make new keys admin-key.pem with  
```
ssh-keygen
```
add it to ssh-agent 
```
ssh-add admin-key.pem
```

**govc: ServerFaultCode: A specified parameter was not correct:**

```
vagrant@vagrant:~/kubernetes$ cluster/kube-up.sh
... Starting cluster using provider: vsphere
... calling verify-prereqs
... calling kube-up
Starting master VM (this can take a minute)...
govc: ServerFaultCode: A specified parameter was not correct:
vagrant@vagrant:~/kubernetes$ govc about
Name:         VMware vCenter Server
Vendor:       VMware, Inc.
Version:      6.0.0
Build:        3018524
OS type:      win32-x64
API type:     VirtualCenter
API version:  6.0
Product ID:   vpx
UUID:         6df8c003-0655-4954-b61d-de3ab0bbe336
vagrant@vagrant:~/kubernetes$
```

```
export GOVC_URL='iskiridomov:****@vcenter'
export GOVC_INSECURE=1 # If the host above uses a self-signed cert
export GOVC_DATASTORE='datastore1'
export GOVC_DATACENTER='*****'
export GOVC_RESOURCE_POOL='*/Resources/Kubernetes'
export GOVC_NETWORK='VM Network'
export KUBERNETES_PROVIDER=vsphere

```

**Comment by mikedanese** [comment]
@imkin

**Comment by imkin** [comment]
@ivan-exness unfortunately vsphere support for kubernetes 1.1 is broken. 

We have fixed it for kubernetes 1.2 release. You could run this on 1.2 beta release being shipped this weekend.
